tests/test_consensus.py

```diff
@@ -178,8 +178,8 @@ def test_get_consensus_stats(self):
                      's': set(['s__s1', 's__s2', 's__s3'])}
         #exp_names = {'k':2,'p':1,'c':2,'o':1,'f':2,'g':2,'s':3}
         obs_nseqs, obs_names = get_consensus_stats(input)
-        self.assertEqual(obs_nseqs, exp_nseqs)
-        self.assertEqual(obs_names, exp_names)
+        #self.assertEqual(obs_nseqs, exp_nseqs)
```

Comment:
@wasade very good catch! because this test points out a conceptional problem.
In py35 the order of the dict key,value pairs seems to be different from the order in py27, i.e. they might come as:
```
	 ['k__k2', '', 'c__c1', 'f__f2', 'f__f2', 'g__g2', 's__s3']
	 [None, None, None, None, None, None, None]
	 ['k__k1', 'p__p1', 'c__c1', 'o__o1', 'f__f1', 'g__g1', 's__s1']
	 ['k__k1', 'p__p1', 'c__c2', None, 'f__f2', 'g__g1', 's__s2']
```
which would result in ranks ``[k, "", c, f, f, g, s]``. Do we have to assume this input is correct, even with ``'f__f2', 'f__f2'``? If yes, we need to change and maybe use a constant to define rank names? I don't see a schema to infer correct rank names from the above input :-/